src/interface/interface.cpp

```diff
@@ -61,34 +61,32 @@ extern "C" void initialize(std::string input_file, int& problem_id,
                            std::vector<std::string>& block_names,
                            std::vector<std::string>& variable_names);
 
-extern "C" void set_external_step_size(const int problem_id,
+extern "C" void set_external_step_size(int problem_id,
```

Comment:
Why did you remove a couple of `const`?

Reply:
An int is passed by value so you don't need to `const` qualify it, doesn't make a difference to the caller.